lib/ansible/modules/cloud/amazon/lightsail.py

```diff
@@ -14,21 +14,24 @@
 DOCUMENTATION = '''
 ---
 module: lightsail
-short_description: Create or delete a virtual machine instance in AWS Lightsail
+short_description: Manage instances in AWS Lightsail.
```

Comment:
Slightly strange convention, description *should* have a full stop at the end, short_description shouldn't.

Reply:
```suggestion
short_description: Manage instances in AWS Lightsail
```